src/server/protocol.ts

```diff
@@ -2040,27 +2040,17 @@ namespace ts.server.protocol {
         languageServiceEnabled: boolean;
     }
 
-    export type ProjectChangedEventName = "projectChanged";
-    export interface ProjectStructureChangedEvent extends Event {
-        event: ProjectChangedEventName;
-        body: ProjectChangedEventBody;
+    export type ProjectsUpdatedInBackgroundEventName = "projectsUpdatedInBackground";

```

Comment:
How will this effect current consumers of the `"projectChanged"` event?

Reply:
Currently there was no event.. This event will be sent on project updating, so that the error diagnostic checks can be started (this is currently happening with contextEvent but that event was not sent correctly or sent always)